broker/staticconfiguration/api.go

```diff
@@ -21,6 +21,7 @@ type BaseConfig struct {
 }
 
 type GitDatabaseConfig struct {
+	Branch                   string        `yaml:"branch"`
```

Comment:
mixing two changesets here?

Reply:
No, because the `gitdb` implementation in the `golib` repository adds a `branch` parameter to the API which the `gitdb` implementation in this repository does not have.